tests/storage/general/test_storage_behavior.py

```
@@ -0,0 +1,87 @@
+        size=size,
+        storage_class=storage_class_name_scope_function,
+    ) as dv:
+        dv.wait_for_dv_success(timeout=TIMEOUT_5MIN)
+        with create_vm_from_dv(
+            vm_name="cnv-3065-vm",
+            client=unprivileged_client,
+            dv=dv,
+            os_flavor=OS_FLAVOR_FEDORA,
+            memory_guest=Images.Fedora.DEFAULT_MEMORY_SIZE,
+        ) as vm_dv:
+            with console.Console(vm=vm_dv) as vm_console:
```

Ruff (0.15.10)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)